src/NuGet.Core/NuGet.Build.Tasks/NuGet.props

```diff
@@ -38,4 +38,14 @@ Copyright (c) .NET Foundation. All rights reserved.
     <CentralPackageVersionsFileImported>true</CentralPackageVersionsFileImported>
   </PropertyGroup>
 
+  <!--
+    Setup default NuGet property values.
+    Visual Studio's project property page requires defaults to be set to inform customers what the default values are.
+  -->
+  <PropertyGroup>
```

Comment:
I'm guessing these defaults in NuGet.props instead of NuGet.targets is just "the right thing to do". 
Any other considerations? Anything related to ordering?

Reply:
The way I see it, the empty check is more appropriate for the SDK changes (including nuget) rather than the project file. 

If you just set that property to something, it'd be respected so isn't that all that matters? 

Can you think of scenarios where someone would want to do that?
I can't really come up with anything right now.

Reply:
I talked to Drew, and [project-system uses MSBuild's `DefaultValueSourceLocation.AfterContext`](https://github.com/dotnet/project-system/blob/3d36410406c3803df33ca7c0fec40d67de9c048a/src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/PageItemRuleProvider.cs#L33) to detect when a project changes a property from its default value, so that the project properties page can show that it's a non-default value.

So, I pushed an update moving the properties to NuGet.targets